rusoto/core/src/region.rs

```diff
@@ -38,6 +38,9 @@ use std::str::FromStr;
 /// `CnNorth1` is currently untested due to Rusoto maintainers not having access to AWS China.
 #[derive(Clone, Debug, PartialEq)]
 pub enum Region {
+    /// Region that covers the Easter part of Asia Pacific
```

Comment:
Looks like `n` was missed in `Eastern`. Should be the only thing to fix before merging this, thanks!